### ISSUE TITLE
Migration: VPN accounts not visible if username contains uppercase letters

### ISSUE THREAD

**Issue by gsanchietti** [NethServer/nethsecurity#966]
**Steps to reproduce**
1. Start with a NethServer 7.9 connected to an external Active directory, where VPN user accounts are correctly recognized and displayed with profiles in lowercase.
2. Perform a migration to NethServer 8.
3. List the users in the remote database with the command:
   ```sh
   root@fwdavtest:~# uci show users
   users.main=local
   users.main.description='Local users'
   users.ns_ldap1=ldap
   users.ns_ldap1.user_attr='sAMAccountName'
   users.ns_ldap1.bind_dn='ldapservice@adns8.nethlab.it'
   users.ns_ldap1.user_bind_dn='%u@adns8.nethlab.it'
   users.ns_ldap1.schema='ad'
   users.ns_ldap1.uri='ldap://rl94ad.adns8.nethlab.it'
   users.ns_ldap1.bind_password='vtiptlOuOFz+v1GLYnjX/lf0PJSEn-sG'
   users.ns_ldap1.base_dn='DC=adns8,DC=nethlab,DC=it'
   users.ns_ldap1.user_dn='DC=adns8,DC=nethlab,DC=it'
   users.ns_ldap1.user_display_attr='displayName'
   users.ns_ldap1.tls_reqcert='never'
   users.ns_ldap1.start_tls='1'
   users.ns_7667497d=user
   users.ns_7667497d.name='test2dav'
   users.ns_7667497d.database='ns_ldap1'
   users.ns_7667497d.openvpn_enabled='1'
   users.ns_231eb50f=user
   users.ns_231eb50f.name='test3dav'
   users.ns_231eb50f.database='ns_ldap1'
   users.ns_231eb50f.openvpn_enabled='1'
   ```
   Confirm that both `test2dav` and upper case `test3dav` are present.
4. On the VPN page, only `test2dav` isvisible.

**Expected behavior**
- Both `test2dav` and `TEST3DAV` should be visible in the VPN profiles section of the web interface.

**Actual behavior**
- Only `test2dav`, which had a lowercase username, is visible in the VPN profiles section, even while the certificate files exist for both users:
  ```sh
  root@fwdavtest:~# ls -hal /etc/openvpn/ns_roadwarrior1/pki/issued/
  drwx------    2 root     root        3.4K Dec  9 16:41 .
  drwx------    7 root     root        3.4K Dec  9 16:42 ..
  -rw-------    1 nobody   nogroup     1.4K Dec  9 16:41 server.crt
  -rw-------    1 nobody   nogroup     1.5K Dec  9 16:41 test2dav.crt
  -rw-------    1 nobody   nogroup     1.5K Dec  9 16:41 test3dav.crt
  ```

**Components**
NethSecurity version: 8-23.05.5-ns.1.3.0 

**References**
- https://helpdesk.nethesis.it/a/tickets/183091

**Comment by gsanchietti** [comment]
**Test case**

Check the issue is not reproducible

**Comment by francio87** [comment]
Verified ✅ 

![Image](https://github.com/user-attachments/assets/b89f3f7c-c7af-4083-ae1a-e93ec559dceb)

**Comment by gsanchietti** [comment]
Released on 23.05.5-ns.1.4.0
